src/components/Admin/ManageSensors/SensorTable/ErrorTags.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  Popover,
+  PopoverTrigger,
+  Button,
+  Portal,
+  PopoverArrow,
+  PopoverContent,
+  PopoverHeader,
+  PopoverCloseButton,
+  PopoverBody,
+  Text,
+  Heading,
+  Flex,
+} from '@chakra-ui/react';
+import {WarningTwoIcon} from '@chakra-ui/icons';
+
+/**
+ * Interface for props for ErrorTag
+ * - `name` - name of the error
+ * - `explanation` - explanation of the error
+ */
+interface ErrorTagProps {
+  name: string;
+  explanation: string;
+}
+
+/**
+ * Tag for an error for the sensor table. When clicked on, a popover appears
+ * with an explanation of the error.
+ * @param name - name of the error
+ * @param explanation - explanation of the error
+ * @returns tag for an error to be displayed in the sensor table. When clicked, the tag will show a popover with an explanation of the error.
+ */
+const ErrorTag: ({name, explanation}: ErrorTagProps) => JSX.Element = ({
+  name,
+  explanation,
+}: ErrorTagProps) => {
+  return (
+    <Popover>
+      <PopoverTrigger>
+        <Button colorScheme="red" size="xs">
+          {name}
+        </Button>
+      </PopoverTrigger>
+      <Portal>
+        <PopoverContent>
+          <PopoverArrow />
+          <PopoverHeader>
+            <Flex alignItems="center" justifyContent="center" marginTop="1em">
+              <WarningTwoIcon color="red.500" />
+              <Heading size="md" color="red.500">
+                {name}
+              </Heading>
+              <WarningTwoIcon color="red.500" />
```

Comment:
Nit: consider adding a space in between the warning icon and the heading.

Reply:
There's one tag in the sensor reading that goes onto a second line and it doesn't look good imo with the warning icons.

Reply:
For some reason it seems to ignore the spaces but I'll try adding a spacer object or something

Reply:
I used an HStack and this worked well